<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GasPricing</class>
 <widget class="QWidget" name="GasPricing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>231</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <widget class="QComboBox" name="bidUnits"/>
   </item>
   <item row="6" column="0" colspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="askUnits"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label5_3">
     <property name="text">
      <string>&amp;Bid: The minimum grice of gas that is accepting into a block which we mine.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="buddy">
      <cstring>bidValue</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QSpinBox" name="bidValue">
     <property name="suffix">
      <string/>
     </property>
     <property name="maximum">
      <number>430000000</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label5_2">
     <property name="text">
      <string>&amp;Ask: The minimum grice of gas that is accepting into a block which we mine.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="buddy">
      <cstring>askValue</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSpinBox" name="askValue">
     <property name="suffix">
      <string/>
     </property>
     <property name="maximum">
      <number>430000000</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
